.github/workflows/build-gpu.yml

```diff
@@ -0,0 +1,45 @@
+name: Build GPU
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+
+    runs-on: [self-hosted, gpu]
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
```

Comment:
In the meantime we have also added 3.9 support

Reply:
Yep, adding.